src/MahApps.Metro/Styles/Controls.Calendar.xaml

```diff
@@ -291,7 +292,7 @@
                                        FontWeight="Bold"
                                        Foreground="{DynamicResource MahApps.Brushes.ThemeForeground}"
                                        Opacity="0.8"
-                                       Text="{Binding}" />
+                                       Text="{Binding FallbackValue={x:Static system:String.Empty}}" />
```

Comment:
I don't think that we need this here, because it's a normal Binding.

Reply:
It is causing the following message:, with the fallback value it disappears,

System.Windows.Data Information: 10 :
Cannot retrieve value using the binding and no valid fallback value exists; using default instead. BindingExpression:(no path); DataItem=null; target element is 'TextBlock' (Name=''); target property is 'Text' (type 'String')

Reply:
@oriches Most of these binding errors are harmless and already handled internally. So we should only look at critical and error level for Fallback values.

Reply:
In my experience, these messages cause any animated transition to stall / jitter, and in general make the UI less smooth

Reply:
I agree with you only partially here. Not all Data Binding outputs mean performance problems. I would only look at warning and error outputs here. Otherwise we would have to do these things in the original WPF styles as well.

![2022-02-09_23h27_47](https://user-images.githubusercontent.com/658431/153301582-91d05182-54d0-4e55-a0f1-51ef25a4ca05.png)